packages/nodejs/nodejs.0.9/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+homepage: "https://github.com/fxfactorial/ocaml-nodejs"
+bug-reports: "https://github.com/fxfactorial/ocaml-nodejs/issues"
+license: "BSD-3-clause"
+dev-repo: "https://github.com/fxfactorial/ocaml-nodejs.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: ["ocamlfind" "remove" "nodejs"]
+depends: [
+  "js_of_ocaml" {>= "2.6"}
+  "oasis" {build & >= "0.4"}
+  "ocamlfind" {build}
+  "yojson"
+  "magic-mime"
+]
+depexts: [
+  [["debian"] ["nodejs"]]
+  [["homebrew" "osx"] ["nodejs"]]
+  [["ubuntu"] ["nodejs"]]
+]
+available: [ocaml-version >= "4.02.3"]
```

Comment:
We'd like to have as broad constraints as possible in most dependency requirements so that it is the package rather than the opam metadata restricting what opam will install. Because opam operates based on metadata that is not directly derived from package properties, we are cautious when it comes to over-constraint. If, in the future, someone tries to install your package on OCaml 4.02.1 and is disallowed by opam, we would be sad if this constraint were artificial. The user may not even realize that they should attempt to relax the constraint in opam to attempt this (perhaps rare) installation.

You're of course welcome to restrict your package to certain OCaml versions or fix your dependency versions very tightly but we'd really encourage you to try to make your constraints as broad as possible to improve the utility of your package and opam as a whole.

Thank you! :smile: